cypress.json

```diff
@@ -0,0 +1,7 @@
+{
+  "baseUrl": "http://localhost:9000",
+  "integrationFolder": "cypress/e2e",
+  "viewportWidth": 1440,
+  "viewportHeight": 768,
+  "video": false
+}
```

Comment:
This file is created to customize the port, and GUI for Cypress. It also can point to different testing directories... so if you wanted to run for certain spaces you can rename the integration folder. I like e2e over integration